extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js

```diff
@@ -4543,16 +4603,15 @@ var globalRequire = require;
 					switch (adEvent.type)
 					{
 						case eventType.LOADED:
+							_resetUIContainerStyle();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the declaration of "_resetUIContainerStyle" before this usage. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)